src/queries/UpdateConnectionQueries.ts

```diff
@@ -0,0 +1,103 @@
+import {Links, ProjectElements, ProjectLinks, Schemes, VocabularyElements} from "../config/Variables";
+import {LinkType} from "../config/Enum";
+import {parsePrefix} from "../function/FunctionEditVars";
+import {qb} from "./QueryBuilder";
+import {LinkConfig} from "../config/LinkConfig";
+import {DELETE, INSERT} from "@tpluscode/sparql-builder";
+
+export function updateDefaultLink(id: string): string {
+	let iri = ProjectElements[ProjectLinks[id].source].iri;
+	let contextIRI = Schemes[VocabularyElements[iri].inScheme].graph;
+
+	let del: string = DELETE`${qb.g(contextIRI, [
```

Comment:
e.g. for this query, it might be enough and better testable to have a string SPARQL template and using BIND/VALUES to provide variable bindings. But maybe at some other place you need to construct the queries dynamically for which the query builder is a good choice.